assets/app/view/create_game.rb

```diff
@@ -45,10 +45,12 @@ def render_content
           }
         end
 
+        inputs << h(:br)
```

Comment:
Please don’t. Use `display: block` or something else.

Reply:
It's already changed. Why did you ask for this?

Reply:
What are you talking about?
I reviewed your changes. You force-pushed the commit removing this line half an hour later.

Reply:
It’s just really bad style. `br` should only be used for texts (in p) and even that’s not a very good idea because of screen-readers.